specification/specs/x_pack/license/start_basic_license/StartBasicLicenseResponse.ts

```diff
@@ -18,7 +18,7 @@
  */
 
 class StartBasicLicenseResponse extends AcknowledgedResponseBase {
-  acknowledge: Dictionary<string, string[]>
+  acknowledge: Dictionary<string, Union<string, string[]>>
```

Comment:
Please avoid using the `Union` type and use the union syntax :)
```suggestion
  acknowledge: Dictionary<string, string | string[]>
```

Reply:
My bad! Be a week or so since I touched this and that dropped out of my head already! Will commit a fix and push regen shortly.